tests/particles/generators_07.cc

```diff
@@ -0,0 +1,120 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#include <deal.II/base/bounding_box.h>
```

Comment:
We generally have a comment here (2-3 lines) that explains what the test does and which function it checks. This makes it easier later on to figure out what a test is about if it fails for some reason.

Reply:
It appears that it is the map_dofs_to_support_points that is not instantiated for the <1,3> case. The compilation error for test generator_07 and generator_08 is the following:

'void dealii::DoFTools::map_dofs_to_support_points<1, 3>(dealii::Mapping<1, 3> const&, dealii::DoFHandler<1, 3> const&, std::map<unsigned int, dealii::Point<3, double>, std::less<unsigned int>, std::allocator<std::pair<unsigned int const, dealii::Point<3, double> > > >&, dealii::ComponentMask const&)'

Reply:
I've now addressed this in #9211. Can you give the additional instantiation a try once #9211 is merged?